packages/adp-tooling/test/unit/writer/__snapshots__/index.test.ts.snap

```diff
@@ -463,10 +491,19 @@ specVersion: \\"3.0\\"
 metadata:
   name: my.test.app
 type: application
+resources:
+  configuration:
+    propertiesFileSourceEncoding: UTF-8
```

Comment:
@korotkovao Can you add a test to write safemode to ui5.yaml?

Reply:
added, thank you @devinea